package/boot/uboot-layerscape-armv8_32b/Makefile

```diff
@@ -8,18 +8,18 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=uboot-layerscape-armv8_32b
-PKG_VERSION:=v1.0
+PKG_VERSION:=2017.03
 PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
-PKG_MIRROR_HASH:=a29edbeddde4b3007ac349c2a5ee434719049ac4753e459dbb1bab21e1c585da
-PKG_SOURCE_URL:=https://github.com/fsl-jyt/uboot-ls-32b.git
-PKG_SOURCE_VERSION:=6476195551ea83193afd06d0c441483c04729c8d
+PKG_MIRROR_HASH:=e8a3294cae1aa0b47fbe159f75051060ab2a81e0a52f7609f5306bd65428f636
+PKG_SOURCE_URL:=git@github.com:yangbolu1991/u-boot-lede.git
```

Comment:
What I mean is:
`https://github.com/yangbolu1991/u-boot-lede.git` instead of `git@github.com:yangbolu1991/u-boot-lede.git`.
The first one allows public download, the second one requires a ssh key for github access.